nautobot/docs/development/core/local-k8s.md

```diff
@@ -0,0 +1,126 @@
+# Using Local Kubernetes Cluster
+
+This guide will help you set up a local Kubernetes cluster using [Kind](https://kind.sigs.k8s.io/) and connect running Nautobot containers to the Kind network.
+
+## Kind Installation
+
+To install Kind on macOS:
+
+```bash
+brew install kind
+```
+
+To install Kind on Linux:
+
+```bash
+export KIND_VERSION=0.24.0
```

Comment:
Is this something we'll need to keep up-to-date over time? Is there a `latest` alias or equivalent that we could document instead of a hard-coded version, maybe similar to how `kubectl` is being handled below?

Reply:
Updated to fetch the latest Kind version.